index.html

```diff
@@ -632,7 +643,19 @@ <h4>The <code>displaystyle</code> and <code>scriptlevel</code> attributes</h4>
             and <code>false</code> and 2, respectively.
           </div>
         </section>
-      </section>
+        <section id="other-valid-attributes">
+          <h4>Attributes Reserved as Valid</h4>
+          <p>The attributes
+          <code>intent</code> and <code>arg</code>
+	  are reserved as valid attributes.</p>
+          <p>While MathML Core Level 1 defines no behavior for these
+          attributes, and no functionality is currently implied by
+          this specification, validators must treat them as
+          valid. Future versions of this specification may or may not
+          define them. Authors should be aware that they are currently
+          in development and subject to change.
+	  </p>
```

Comment:
Similarly, I would say that MathML Core does not define any behavior for `intent` and `arg` and put the rationale for including these attributes in a (non-normative) note.

Reply:
as above, I'll adjust